scripts/install-maria.sh

```diff
@@ -1,14 +1,15 @@
 
 # Check If Maria Has Been Installed
 
+set -e
+set -x
```

Comment:
Yeah it's great for debugging, but to the precedent set in other places in homestead is not to output unless there's a failure.